Makefile

```diff
@@ -179,6 +179,10 @@ lint:
 unit:
 	hack/unit.sh ${GO_MOD_FLAGS}
 
+.PHONY: community
+community:
+	hack/community/generate.sh
```

Comment:
The required arguments for the script are missing here

Reply:
Adding those next - I have to rework the script